docs/content/quickstart/build-bundle.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Build your Bundle
```

Comment:
I am concerned that this goes into too much detail for a quickstart. 

A goal of Porter is that first-time users should make bundles without editing or reading the generated Dockerfile, knowing of the existence or contents of the .cnab directory, etc. This is putting all that knowledge up front in the quickstart and gives the impression that understanding that information is necessary. 

It seems overwhelming to me, and gives equal weight to topics that are not as important as others which are critical to using Porter successfully such as parameters, credentials, etc.